java/src/org/openqa/selenium/edge/BUILD.bazel

```diff
@@ -17,5 +18,6 @@ java_export(
         "//java/src/org/openqa/selenium:core",
         "//java/src/org/openqa/selenium/chromium",
         "//java/src/org/openqa/selenium/remote",
+        artifact("com.google.guava:guava"),
```

Comment:
:(

Reply:
It's fine, but at some point, we'll want to remove guava as a project dep as it's pretty heavy.

Reply:
I just kept pulling levers until bazel was happy. I do not know what actually needed this, but adding this line made it work. :)